coucharchive

```diff
@@ -498,10 +554,12 @@ def load(filename):
     _load_archive(filename, callback)
 
 
-def restore(target, filename, reuse_db_if_exist=False, ignore_dbs=[]):
+def restore(target, filename, reuse_db_if_exist=False,
+            queue_control=True, ignore_dbs=[]):
     def callback(local_couch_server_url):
         replicate_couchdb_server(local_couch_server_url, target,
                                  reuse_db_if_exist=reuse_db_if_exist,
+                                 queue_control=queue_control,
```

Comment:
This one is not very important, it's more a cherry on the cake... but I propose to use `no_run_queue_control` or `has_run_queue_control` in Python arguments. Otherwise, when I read this, it looks like you're passing an object.

Reply:
Clever, I will use `has_run_queue_control` if it's OK.